packages/core/src/model.js

```diff
@@ -2334,7 +2334,7 @@ ${associationOwner._getAssociationDebugList()}`);
               logging: options.logging,
             })
```

Comment:
```suggestion
              logging: options.logging,
              ignoreDuplicates: options.ignoreDuplicates
            })
```

You can also do this instead